stubs/fpdf2/fpdf/fpdf.pyi

```diff
@@ -179,7 +200,7 @@ class FPDF:
     def rect(self, x, y, w, h, style: Any | None = ...) -> None: ...
     def ellipse(self, x, y, w, h, style: Any | None = ...) -> None: ...
     def circle(self, x, y, r, style: Any | None = ...) -> None: ...
-    def add_font(self, family: str, style: _FontStyle = ..., fname: str | None = ..., uni: bool = ...) -> None: ...
+    def add_font(self, family: str, style: _FontStyle = ..., fname: StrPath | None = ..., uni: bool = ...) -> None: ...
```

Comment:
It does `if fname.endswith(".ttf"):`, so I think this has to stay str. The other branch also does `Path / fname`.

Reply:
```suggestion
    def add_font(self, family: str, style: _FontStyle = ..., fname: str | None = ..., uni: bool = ...) -> None: ...
```